mac/KeymanEngine4Mac/KeymanEngine4MacTests/KMEngineTests.m

```diff
@@ -585,8 +593,8 @@ - (void)testCoreProcessEvent_eventSingleQuoteWithElNuerKmx_ReturnsDiacritic {
     NSEvent *event = [NSEvent keyEventWithType:NSEventTypeKeyDown location:NSMakePoint(0, 0) modifierFlags:0 timestamp:0 windowNumber:0 context:nil characters:@"'" charactersIgnoringModifiers:@"'" isARepeat:NO keyCode:kVK_ANSI_Quote];
     CoreKeyOutput *output = [engine processEvent:event];
     XCTAssert(output.hasTextToInsert, @"returns text to insert");
-    context = engine.getCoreContext;
-    XCTAssert([context isEqualToString:@"\u025B\u0308"], @"Context updated with diacritic.");
+    context = engine.getCoreContextDebug;
+    XCTAssert([context isEqualToString:@"|\u025B\u0308| (len: 2) [ U+025b U+0308 ]"], @"Context updated with diacritic.");
```

Comment:
Is `getCoreContextDebug` in the unit tests going to be long-term?

Could replace `\u025B\u0308` with `ɛ̈`

Reply:
> Is `getCoreContextDebug` in the unit tests going to be long-term?

Well for the foreseeable future, yes. Why?

> Could replace `\u025B\u0308` with `ɛ̈`

I think it's clearer to be able to see the combining diacritic is a separate codepoint, this way.

Reply:
Gotcha. I just found it nice in CoreWrapperTests.m to see the emoji characters and corresponding codepoints.